E2E/pageObjects/game.page.ts

```diff
@@ -98,12 +102,31 @@ export = {
     I.switchToPreviousTab();
     I.seeElement(locator.chessBoard.board);
   }, 
+
   skipMove: () => {
     I.click(locator.buttons.skip);
     I.seeElement(locator.buttons.skipButtonHighlighted);
   },
+
   voteAndCheckThatVoteIsVisible:(chessPiece: ChessPiece, tile: string, value:string) => {
     game.vote(chessPiece, tile);
     game.voteIsVisible(chessPiece, tile, value);
   },
+
+  checkPlayerSP:(averageScore) => {
```

Comment:
When rounding up the average score, if delta is the same to lower and higher fibonacci number, it should return the higher number, your method does the opposite. For example, if `averageScore` is 4, it should return 5, but test returns 3.